credentials/credentials.go

```diff
@@ -101,6 +101,8 @@ type ProtocolInfo struct {
 	// SecurityProtocol is the security protocol in use.
 	SecurityProtocol string
 	// SecurityVersion is the security protocol version.
```

Comment:
Should we update this comment to explain that it's a static version string, not computed per-connection, and consequently has little value?

Reply:
Updated the comment.